examples/01_tutorials/plot_first_level_model_details.py

```diff
@@ -197,9 +197,9 @@ def plot_contrast(first_level_model):
 # 
 # By default the drift model is a set of slow oscillating functions (Discrete Cosine transform), with a cutoff at frequency 1/128 hz.
 # We can change this cut-off, e.g. to 1/64Hz.
-# This is done by setting period_cut=64(s)
+# This is done by setting high_pass=0.01(Hz)
```

Comment:
high_pass should be set to the value of 1./64 which is 0.016 . 
We need to tweak the language in these examples.

Reply:
No, I'd take 0.01 hz, which is more standard I think

Reply:
I've kept 0.016. According to the comment of the example (lines 198 and 199), it is stated that the default value of high_pass is 1/128Hz, i.e. 0.01Hz, but it can be changed to e.g. 1/64Hz, i.e. 0.016.